integration-tests/fixtures/generate-bundle-stats/astro/astro-integration.test.ts

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { $ } from "bun";
+import { describe, it, expect, afterEach, beforeEach } from "bun:test";
+import { GenerateConfig } from "../../../scripts/gen-config";
+
+const astroApp = "test-apps/astro";
+
+const VERSIONS = [4];
+
+const FORMATS = [
+  { format: "es", expected: "esm" },
+  { format: "esm", expected: "esm" },
+  { format: "module", expected: "esm" },
+];
+
+describe("Generating astro stats", () => {
+  describe.each(VERSIONS)("%d", (version) => {
+    describe.each(FORMATS)("%o", ({ format, expected }) => {
+      beforeEach(async () => {
+        const config = new GenerateConfig({
+          // need to re-write config script so that it can point to astro dir but still output the astro config file
```

Comment:
> // need to re-write config script so that it can point to astro dir but still output the astro config file

On this note, until we do this, what happens instead?

Reply:
Whoops, left over comment, not needed